Gruntfile.js

```diff
@@ -21,6 +23,17 @@ module.exports = function(grunt) {
 
   grunt.config.set('compress', require('./grunt/config/compress'));
 
+  function spawnGulp(args, opts, done) {
+    grunt.util.spawn({
+      cmd: 'gulp',
```

Comment:
Can this be node_modules/gulp/bin/gulp.js? I don't have gulp installed globally.